tests/auto/rt_auto.py

```diff
@@ -0,0 +1,474 @@
+# HEADER
+# Written by Brian Curtis
+# Automation of RT tasks using github cli
+from __future__ import print_function
+
+from github import Github as gh
+import datetime
+import time
+import pandas as pd
```

Comment:
why is Pandas needed here? Is there any "data analysis" being done in this script that submits and runs jobs?

Reply:
Are you saying use something else completely or `from pandas import DataFrame` instead importing all of pandas?

Reply:
why do you want to add pandas dependency to this script?

Reply:
I've imported just the DataFrame for now, what's wrong with Pandas?

Reply:
if you were doing data analysis, nothing.
You just need a table or a dictionary.